waspc/src/Wasp/AI/GenerateNewProject.hs

```diff
@@ -56,14 +57,18 @@ generateNewProject newProjectDetails waspProjectSkeletonFiles = do
           <> showParams (codingChatGPTParams newProjectDetails)
           <> " for coding."
 
-  writeToLogGenerating "project skeleton..."
-  (waspFilePath, planRules) <-
-    generateAndWriteProjectSkeletonAndPresetFiles newProjectDetails waspProjectSkeletonFiles
+  writeToLogGenerating "project's initial files..."
+  IF.IniitalFilesGenResult
+    { IF._waspFilePath = waspFilePath,
+      IF._prismaFilePath = prismaFilePath,
+      IF._planRules = planRules
+    } <-
+    IF.genAndWriteInitialFiles newProjectDetails waspProjectSkeletonFiles
   writeToLog "Generated project skeleton."
```

Comment:
The name `waspProjectSkeletonFiles` is okay since it references only the skeleton files.

One of the params for `genAndWriteInitialFiles` is the `waspProjectSkeletonFiles` :)

I've update the log 👍